packages/react-pdf-toolkit/src/utils/get-risk-score-style.ts

```diff
@@ -0,0 +1,17 @@
+export const getRiskScoreStyle = (score: number | null = 0) => {
+  score = score === null ? 0 : score;
```

Comment:
Should not make a mutation to the received score especially in the context of React and risk score.

Reply:
P0

Reply:
Its not mutation, its default for nullish value. We always have to show score even if it not present.

Reply:
Don't assign function arguments.